src/NewPost.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from "react";
+import { data } from "./data";
+
+export class NewPost extends Component {
+  state = {
+    id: 0,
+    title: "",
+    author: "",
+    createdOn: "",
+    body: "",
+    tags: ""
+  };
+
+  createDateOnSubmit = () => {
```

Comment:
there's a sneaky little bug with this function. i just made a new post in my browser and it got `createdOn: "2019-1-27"`. what's wrong with that?

Reply:
yes. months by default are 0-11. I